spec/models/automated_checks/claim_verifiers/identity_spec.rb

```diff
@@ -104,6 +104,12 @@ module ClaimVerifiers
 
             before { perform }
 
+            describe "#claim_verifier_match" do
+              subject(:claim_verifier_match) { identity_confirmation_task.claim_verifier_match }
+
+              it { is_expected.to eq "claim_verifier_match_all" }
```

Comment:
@nomasprime as per the enums comment, could this not become just `eq all`?

Reply:
No, needs to be prefixed. Can't define `all` on a Rails model.